OpenRA.Mods.Common/Effects/SpriteEffect.cs

```diff
@@ -48,7 +48,7 @@ public class SpriteEffect : IEffect, ISpatiallyPartitionable
 			this.visibleThroughFog = visibleThroughFog;
 			this.delay = delay;
 			pos = posFunc();
-			anim = new Animation(world, image, facingFunc);
+			anim = new Animation(world, image, () => WAngle.FromFacing(facingFunc()));
```

Comment:
This gets reverted in https://github.com/OpenRA/OpenRA/pull/18058/commits/c77e885a7e55449ee20dadfe924ea1122f47dced again.

Reply:
There are far too many changes to do in one commit or one PR, so the idea is to go step by step on focused parts of the code. In these early steps this means adding a lot of conversions that will be removed later - sometimes in the next commit.